CONTRIBUTING.md

```diff
@@ -104,7 +104,9 @@ You can specify the special `test` attribute in the configuration that will not
 
 - `test.runNvim`: Set to `false` to avoid launching nvim with this configuration and simply build the configuration.
 
-The tests are then runnable with `nix flake check --all-systems`.
+A single test can be run with `nix develop --command tests --interactive`. This launches the testing suite in interactive mode, allowing you to easily search for and select specific tests to run.
+
+The entire test suite is then runnable with `nix flake check --all-systems`.
```

Comment:
Maybe we should caution against running the full test suite locally?

Most of the time you only really care about a few specific tests, and CI will run the full test suite on any PRs that get opened _anyway_ 🤔

Reply:
<img width="858" alt="Screenshot 2024-11-11 at 7 29 21 PM" src="https://github.com/user-attachments/assets/f14cc048-6374-4096-8229-c9944eb6c8c8">
Added a note about cautioning against running the entire testing suite.